### DISCUSSION TITLE
Proteger codigo Aplicativos

### DISCUSSION THREAD

**Discussion by ESTUQUE** [Sketchware-Pro/Sketchware-Pro#840] [Q&A]
Todo aplicativo que construo atravez de Sketchware invadem facil. Gostaria de saber se ha uma maneira de proteger o apk.

**Comment by j-px** [comment]
You can use ProGuard and StringFog. Just make sure to test your projects after turning both on, some things might break and you might have to customize ProGuard using rules.
